build_projects/Microsoft.DotNet.Cli.Build.Framework/Command.cs

```diff
@@ -293,37 +293,40 @@ public Command OnErrorLine(Action<string> handler)
             return this;
         }
 
-        private string FormatProcessInfo(ProcessStartInfo info)
+        private string FormatProcessInfo(ProcessStartInfo info, bool includeWorkingDirectory)
```

Comment:
I don't know how this change got brought in, this is from one of @eerhardt previous commits, probably to cli, and I brought it over on accident. 

Not going to make changes here.